src/libcore/array.rs

```diff
@@ -57,6 +58,11 @@ unsafe impl<T, A: Unsize<[T]>> FixedSizeArray<T> for A {
     }
 }
 
+/// The error type returned when a conversion from a slice to an array fails.
+#[unstable(feature = "try_from", issue = "33417")]
+#[derive(Debug, Copy, Clone)]
+pub struct TryFromSliceError(());
```

Comment:
This should probably implement `Error` in libstd.

Reply:
Should the `Display` implementation also be in libstd or libcore?

Reply:
See e45e8abec2302530d774aab1ca800081b60aa89e